systemtest/src/test/java/io/strimzi/systemtest/LogSettingST.java

```diff
@@ -245,6 +249,23 @@ void testGcLoggingSetDisabled() {
         assertThat("Mirror-maker GC logging is disabled", checkGcLoggingDeployments(KafkaMirrorMakerResources.deploymentName(CLUSTER_NAME)), is(false));
     }
 
+    @Test
+    @Order(11)
+    void testKubectlGetStrimzi() {
+        String userName = "test-user";
+        String topicName = "test-topic";
+
+        KafkaTopicResource.topic(CLUSTER_NAME, topicName).done();
+        KafkaUserResource.tlsUser(CLUSTER_NAME, userName).done();
+
+        String strimziCRs = cmdKubeClient().execInCurrentNamespace("get", "strimzi").out();
+
+        assertThat(strimziCRs, containsString(CLUSTER_NAME));
+        assertThat(strimziCRs, containsString(GC_LOGGING_SET_NAME));
+        assertThat(strimziCRs, containsString(userName));
+        assertThat(strimziCRs, containsString(topicName));
```

Comment:
is there a plan for some other test to check that even Kafka Bridge, Mirror Maker, Mirror Maker 2 and Connect are there as well?

Reply:
I guess that could be added to the ST for those specific CRs.

Reply:
Well currently for example Bridge is deployed with same name as Kafka and only way how to distinguish it's Kafka and not Bridge could be via some regex. Question is - is that worthy? Or change the name of each component and assert all new names. I can try both later (there are other things with higher priority atm).

Reply:
Why the bridge is deployed with the same name as Kafka? Even our examples have `my-cluster` and `my-bridge` as different names.

Reply:
I think I just wanted to use only one variable. I will change it in this PR.